ftrackhq/ftrack-recipes
# chore: updated implementation of the cascade thumbnail updater

Resolves  PSFT-8a06116c-5517-11ed-a985-2646f76b5e2e


## Changes

Provide updated code from [old bitbucket snippet](https://bitbucket.org/ftrack/workspace/snippets/ngaro/event-listener-cascade-thumbnails)

## Test
[assumes you have ftrack environment variables set to an instance you can use]

1) ensure a venv with python3.7+ and ftrack-python-api installed
2) run from command line , from the hook folder the script
3) upload a new version containing an image or a movie, the thumbnail will also be updated on the parent entity

note:

you might have to refresh the page to see the new thumbnail on the parent entity.
